<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RPiCarClient</class>
 <widget class="QMainWindow" name="RPiCarClient">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>499</width>
    <height>378</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>RPiCarClient</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QSlider" name="speedSlider">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>50</y>
      <width>51</width>
      <height>221</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QDial" name="angleDial">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>100</y>
      <width>121</width>
      <height>111</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="speedLabel">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>280</y>
      <width>67</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QLabel" name="angleLabel">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>210</y>
      <width>67</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>499</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
